src/Exception/InvalidIfExpressionException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sofascore\PurgatoryBundle\Exception;
+
+use Symfony\Component\ExpressionLanguage\Expression;
+use Symfony\Component\ExpressionLanguage\SyntaxError;
+
+final class InvalidIfExpressionException extends RuntimeException
+{
+    private const MESSAGE = 'Invalid "if" expression: "%s"';
+
+    public function __construct(Expression $expression, SyntaxError $syntaxError)
+    {
+        parent::__construct(
+            message: \sprintf(self::MESSAGE, $expression),
+            previous: $syntaxError,
+        );
+    }
```

Comment:
@Brajk19 Do we want the message to give us a clue:

```suggestion
    private const MESSAGE = 'Invalid "if" expression ("%s") provided: %s';

    public function __construct(Expression $expression, SyntaxError $syntaxError)
    {
        parent::__construct(
            message: \sprintf(self::MESSAGE, $expression, $syntaxError->getMessage()),
            previous: $syntaxError,
        );
    }
```

Reply:
@HypeMC yup, makes sense
https://github.com/sofascore/purgatory-bundle/pull/101/commits/1f4a8a34c9f5dc0dc715500b4c9d7759c43d4917